tiled/server/schemas.py

```diff
@@ -444,4 +445,42 @@ def specs_uniqueness_validator(cls, v):
         return v
 
 
+# we use functional syntax to define JSONPatchSpec since "from" is a keyword
+JSONPatchSpec = TypedDict(
+    "JSONPatchSpec",
+    {
+        "op": str,
+        "path": str,
+        "from": str,
+        "value": Any,
+    },
+    total=False,
+)
+
+
+class PatchMetadataRequest(pydantic.BaseModel):
+    # These fields are optional because None means "no changes; do not update".
+    patch: Optional[Union[List[JSONPatchSpec], Dict]]  # Dict for merge-patch
+    specs: Optional[Specs]
+
+    # Wait for fix https://github.com/pydantic/pydantic/issues/3957
+    # to do this with `unique_items` parameters to `pydantic.constr`.
```

Comment:
Looks like they've fixed it already in ver 1.10.12 (we use 1.10.13). Would it be worthwhile just uncommenting line 86?

Reply:
or maybe in a separate PR?

Reply:
I marked this one as resolved by mistake. This came from a part of code I copied from @danielballan. Not exactly sure what needs to be done here, but I agree maybe it should be a separate PR.

https://github.com/bluesky/tiled/blame/e056aa9694375d2b034d69a0143d40845b6ee2bb/tiled/server/schemas.py#L436-L437